tallycash/extension
# Accounts: New grouping

Closes #2305 

This PR updates the accounts view and makes new grouping for accounts by categories. 

**Changes**

- update icons for Wallet types
- add top level categories for each type of wallets
- remove Tabs and add page title
- remove the rounded corner from the bottom of the slide-up menu

**UI**

Before
<img width="240" alt="Screenshot 2022-09-28 at 10 15 47" src="https://user-images.githubusercontent.com/23117945/192726536-a20ef343-3581-49ac-aff9-1ea1958d412c.png">

<img width="240" alt="Screenshot 2022-09-28 at 10 15 39" src="https://user-images.githubusercontent.com/23117945/192726541-abad91c6-4fa8-4472-bf6b-fc8d42fe7a6d.png">

After

<img width="240" alt="Screenshot 2022-09-28 at 09 20 41" src="https://user-images.githubusercontent.com/23117945/192724709-5a40ea76-9ef8-48a8-9c06-5ea49ec9c1c3.png">

<img width="240" alt="Screenshot 2022-09-28 at 10 08 59" src="https://user-images.githubusercontent.com/23117945/192724904-65e6aec6-1c09-4cf1-af80-ca7b8cd06349.png">

<img width="240" alt="Screenshot 2022-09-28 at 10 09 24" src="https://user-images.githubusercontent.com/23117945/192724982-eccb69da-840c-4bbe-a98f-77c52185a282.png">

**To Test**
Create a several accounts (internal, imported, read-only and from ledger). Go to accounts view. Check how accounts categories are displayed.
- [ ] for all accounts type 
- [ ] when internal accounts don't exist
